forum/users/permissions.py

```diff
@@ -43,6 +43,15 @@ class IsInvestorNamespaceSelected(BaseNamespaceSelectedPermission):
     NAMESPACE = 'investor'
     
     
+class IsNamespace(permissions.BasePermission):
+    def has_permission(self, request, view):
+        payload = get_token_payload_from_cookies(request)
+        namespace = payload.get('name_space_name')
+        if namespace not in ['startup', 'investor']:
```

Comment:
You can use enum https://github.com/Project-Stage-Academy/UA1198ForumSB/blob/main/forum/communications/mongo_models.py#L15

Reply:
Done, thanks